model_compiler/README.md

```diff
@@ -15,17 +15,36 @@ MIVisionX allows hundreds of different [OpenVX](https://www.khronos.org/registry
 <p align="center"><img width="100%" src="../docs/images/runtime.png" /></p>
 
 ## Pre-requisites
-* MIVisionX libraries
+
+* Ubuntu 16.04/18.04 or CentOS 7.5/7.6
+* [MIVisionX libraries](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX#build--install-mivisionx)
```

Comment:
* [MIVisionX](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX#build--install-mivisionx) - Install MIVisionX
**Note:** MIVisionX installs all the model compiler scripts at ```/opt/rocm/mivisionx/model_compiler/python/```